### PR TITLE
Add EmailOTP and TOTP HTML based email templates

### PR DESCRIPTION
**Pull request by AnuradhaSK**
### Proposed changes in this pull request

Resolves https://github.com/wso2/product-is/issues/7079

### Follow up actions

Merge https://github.com/wso2/product-is/pull/7779

### CODE REVIEW

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 1]
```diff
@@ -1606,4 +1606,120 @@
...
+                        <td style="text-align: left; padding: 30px 50px 50px 50px" valign="top">
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #505050; text-align: left;">
+                                Thanks,<br/>WSO2 Identity Server Team
+                            </p>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td colspan="2" align="center" style="padding: 20px 40px 40px 40px;" bgcolor="#f0f0f0">
+                            <p style="font-size: 12px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #777;">
+                                &copy; 2018
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Change the year

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 2]
```diff
@@ -1606,4 +1606,120 @@
...
+                            </p>
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                                Please use this one time password <b>{{OTPCode}}</b> to sign in to your application
+                            </p>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td style="text-align: left; padding: 30px 50px 50px 50px" valign="top">
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #505050; text-align: left;">
+                                Thanks,<br/>WSO2 Identity Server Team
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Shall we add a `.` at the end of the line?

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 3]
```diff
@@ -1606,4 +1606,120 @@
...
+                            </p>
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                                Please use the token <b>{{token}}</b> as the password for your login.
+                            </p>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td style="text-align: left; padding: 30px 50px 50px 50px" valign="top">
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #505050; text-align: left;">
+                                Thanks,<br/>WSO2 Identity Server Team
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Shall we add `.` at the end of the line?

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 4]
```diff
@@ -1606,4 +1606,120 @@
...
+                            </p>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td colspan="2" align="center" style="padding: 20px 40px 40px 40px;" bgcolor="#f0f0f0">
+                            <p style="font-size: 12px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #777;">
+                                &copy; 2018
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Change the year

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 5]
```diff
@@ -1606,4 +1606,120 @@
...
+                            </h1>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td style="text-align: left; padding: 0px 50px;" valign="top">
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                                Hi {{user.claims.givenname}},
+                            </p>
+                            <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                                Please use this one time password <b>{{OTPCode}}</b> to sign in to your application
```
[the remark this answers is not in the document]
Comment (AnuradhaSK) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Updated

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 6]
```diff
@@ -1606,4 +1606,120 @@
...
+                            <a href="http://wso2.com/" target="_blank">
+                                <img src="http://cdn.wso2.com/wso2/newsletter/images/nl-2017/wso2-logo-transparent.png"
+                                     border="0"/>
+                            </a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td colspan="2" align="center" style="padding: 50px 50px 0px 50px;">
+                            <h1 style="padding-right: 0em; margin: 0; line-height: 40px; font-weight:300; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 1em;">
+                                WSO2 IS Email OTP
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:1627]: Shall we change `IS` to `Identity Server`?

**Pull request merged**